frontend/src/locales/en/messages.po

```diff
@@ -61,6 +61,30 @@ msgstr "Verifying OTP..."
 msgid "allowedImageSources"
 msgstr "file.go.gov.sg"
 
+#: src/config.ts:61
+msgid "announcement.imageUrl"
+msgstr "file.go.gov.sg"
+
+#: src/config.ts:62
+msgid "announcement.readGuideUrl"
+msgstr "https://guide.postman.gov.sg/release-notes#2020-11-05-or-demo-campaigns-for-sms-and-telegram"
+
+#: src/config.ts:60
+msgid "announcement.subtext"
+msgstr "We've heard your requests to cater to more channels. Good news, Postman is now supporting agencies to utilise their Telegram bots!  We have chosen to work with Telegram as it is a messaging app with  a focus on speed and security. It's super-fast, simple and free."
```

Comment:
Might need to check with either Petty or the designers what should be the exact content for announcement.